### PR TITLE
Templates [POC]

### LINKED ISSUE #33: New Template system
Using string template literals under the hood ✨ (ala Rails partials):

```js
render('users/form', { name: user.name, back_to: '/users'})
```

Interesting links:
- https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals
- https://developer.mozilla.org/en-US/docs/Web/HTML/Element/template
- https://underscorejs.org/#template
- https://github.com/borismoore/jsrender

### PR DESCRIPTION
**Pull request by markets**
Closes #33 

Just a proof of concept for now. Example here: https://github.com/ralixjs/ralix-todomvc/pull/3

Basically, Ralix templates are js functions with Template literals.

Example
```js
// app/javascript/templates/index.js
export const todoItem = ({ id, label }) => `
  <div id="item_${id}">
    <input type="checkbox" onclick="toggleCheck(${id})">
    <label>${label}</label>
    <button onclick="destroyItem(${id})"></button>
  </div>
`

// call it via
render('todoItem', { id: id, value: value })
```

### CODE REVIEW

**Comment by markets** [comment]
@AlexLarra @lautarocastillo @maguri @jfelip937 @araluce

What do you think team? Pure js functions with template literals ✨

I'd like in future PRs to allow/document splitting templates in files/subfolders (necessary for bigger apps, as using only 1 file would become a mess).

File: README.md [hunk 1]
```diff
@@ -162,6 +161,33 @@ export default class FlashMessages {
...
+export const todoItem = ({ id, label }) => `
+  <div class="item_${id}">
+    <input type="checkbox">
+    <label>${label}</label>
+    <button onclick="destroyItem(${id})"></button>
+  </div>
+`
+
+// Call it via
+render('todoItem', { id: id, value: value })
```
Comment (maguri) [inline README.md:188]: `label` key is missing, should be value in the definition checking the example

File: README.md [hunk 2]
```diff
@@ -162,6 +161,33 @@ export default class FlashMessages {
...
+import * as Templates from 'templates'
+
+const App = new RalixApp({
+  ...
+  templates: Templates
+  ...
+})
+
+// app/javascript/templates/index.js
+export const todoItem = ({ id, label }) => `
```
Comment (maguri) [inline README.md:179]:
```suggestion
export const todoItem = ({ id, value}) => `
```

File: README.md [hunk 3]
```diff
@@ -162,6 +161,33 @@ export default class FlashMessages {
...
+  <div class="item_${id}">
+    <input type="checkbox">
+    <label>${label}</label>
```
Comment (maguri) [inline README.md:182]:
```suggestion
    <label>${value}</label>
```

**Comment by markets** [comment]
Thanks for the review Maurets! 👌 

Take care man! Cheers 👋

**Pull request merged**